drivers/include/periph/flashpage.h

```diff
@@ -106,6 +122,26 @@ static inline int flashpage_page(void *addr)
  */
 void flashpage_write(int page, void *data);
 
+/**
+ * @brief   Write the given address with the given data and length
+ *
+ * @warning Make sure the targeted memory area is erased before calling
+ *          this function
+ *
+ * Both target address and data address must be aligned to
+ * @p FLASHPAGE_RAW_ALIGN. len must be a multiple of @p FLASHPAGE_RAW_BLOCKSIZE.
+ * This function doesn't erase the block to be written automatically,
```

Comment:
I would vote to remove the word 'block' here and refer to 'area in flash' or 'memory area' as you did above, as there could be multiple blocks involved...